boards/arm/nucleo_f070rb/doc/nucleof070rb.rst

```diff
@@ -0,0 +1,179 @@
+.. _nucleo_f070rb_board:
+
+ST Nucleo F070RB
+################
+
+Overview
+********
+The STM32 Nucleo-64 development board with STM32F070RB MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable, and flexible way for users to try out new concepts,
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption, and features.
+
+The Arduino* Uno V3 connectivity support and the ST morpho headers allow easy functionality
+expansion of the STM32 Nucleo open development platform with a wide choice of
+specialized shields.
+
+The STM32 Nucleo board integrates the ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Nucleo board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/nucleo_f070rb_board.jpg
+     :width: 500px
+     :height: 367px
+     :align: center
+     :alt: Nucleo F070RB
+
+More information about the board can be found at the `Nucleo F070RB website`_.
+
+Hardware
+********
+Nucleo F070RB provides the following hardware components:
+
+- STM32 microcontroller in QFP64 package
+- Two types of extension resources:
+    - Arduino* Uno V3 connectivity
+    - ST morpho extension pin headers for full access to all STM32 I/Os
+- ARM* mbed*
+- On-board ST-LINK/V2-1 debugger/programmer with SWD connector:
+    - Selection-mode switch to use the kit as a standalone ST-LINK/V2-1
+- Flexible board power supply:
+    - USB VBUS or external source (3.3V, 5V, 7 - 12V)
+    - Power management access point
+- Three LEDs:
+    - USB communication (LD1), user LED (LD2), power LED (LD3)
+- Two push-buttons: USER and RESET
+- USB re-enumeration capability. Three different interfaces supported on USB:
+    - Virtual COM port
+    - Mass storage
+    - Debug port
+- Support of wide choice of Integrated Development Environments (IDEs) including:
+    - IAR
+    - ARM Keil
+    - GCC-based IDEs
+
+More information about STM32F070RB can be found here:
```

Comment:
This will display better if written as:

    More information about TM32F070RB can be found in
    the `STM32F070 reference manual`_.